openthread_border_router/Dockerfile

```diff
@@ -55,6 +56,7 @@ RUN \
     && ./script/bootstrap \
     && patch -p1 < /usr/src/0001-Avoid-writing-to-system-console.patch \
     && patch -p1 < /usr/src/0002-rest-support-deleting-the-dataset.patch \
+    && patch -p1 < /usr/src/0001-openthread-set-netif-route-metric-lower.patch \
```

Comment:
shouldnt this be 0003 ?

Reply:
It numbering only really matter if the patches touch the same files. I've renamed it anyways, makes it look cleaner.